check-log/check-log.go

```diff
@@ -362,9 +362,9 @@ func writeFileAtomically(f string, contents []byte) error {
 	}
 	defer os.Remove(tmpf.Name())
 	_, err = tmpf.Write(contents)
+	defer tmpf.Close()
```

Comment:
It's ok to call here but no reason to place this line right after L363? ref: https://github.com/golang/tools/blob/be0fcc31ae2332374e800dfff29b721c585b35df/cmd/goimports/goimports.go#L285-L286